all-clouds/all-clouds-how-to/check-cve-on-instance.rst

```diff
@@ -96,13 +97,7 @@ The commands used to scan the instance vary slightly based on whether your insta
 Download the scan report
 ------------------------
 
-You can download the report from the instance to your local machine using the gcloud CLI. Assuming you ran ``oval eval`` in the home directory of a user named "ubuntu_user" on an instance named "ubuntu-instance", the command to download the report is:
-
-.. code::
-
-    gcloud compute scp ubuntu_user@ubuntu-instance:report.html .
-
-You'll find ``report.html`` in the directory where you executed the command, and if you open it in a browser you'll see something similar to: 
+The scan report will be available as ``report.html`` in the home directory of your instance. You can download it to your local machine using some file transfer service such as ``scp``, and if you open it in a browser you'll see something similar to: 
```

Comment:
The first sentence should be something like `The scan report will be available as ``report.html`` in the directory of your instance where you ran ``oval eval`` `

Reply:
Done, thanks!